src/OpenTelemetry.Instrumentation.GrpcNetClient/GrpcClientInstrumentationOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="GrpcClientInstrumentationOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Instrumentation.GrpcNetClient
+{
+    /// <summary>
+    /// Options for GrpcClient instrumentation.
+    /// </summary>
+    public class GrpcClientInstrumentationOptions
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether instrumentation is suppressed (disabled).
+        /// </summary>
+        public bool SuppressInstrumentation { get; set; }
```

Comment:
I guess the intention is to suppress the underlying HTTP instrumentation (for gRPC over HTTP) so we only have gRPC spans? If this is the case, the name needs to reflect the intention.

Reply:
Agree with @reyang. If you are configuring instrumentation, what does "suppress instrumentation" mean? Users are going to be very confused because in their minds, there is just one instrumentation at play. Seems like an on/off switch for the Grpc instrumentation :smile: Maybe something like `DisableTransportSpans`? 🤷